plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -829,6 +858,10 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 				   (guint8 *)&self->dynamic_ts,
 				   sizeof(FuGenesysDynamicToolString));
 	}
+	self->dynamic_ts.ss_port_number =
+	    _fu_genesys_revert_hex_char_to_int(self->dynamic_ts.ss_port_number);
+	self->dynamic_ts.hs_port_number =
+	    _fu_genesys_revert_hex_char_to_int(self->dynamic_ts.hs_port_number);
```

Comment:
ditto

Reply:
I think these value only use to generate guid, I will keep it as a hex value.

Reply:
done.

Reply:
ditto.